internal/controllers/watchdog/controller.go

```diff
@@ -17,6 +18,19 @@ type watchdogController struct {
 	threshold time.Duration
 }
 
+func (c *watchdogController) getInputsExist(comp *apiv1.Composition, ctx context.Context) bool {
+	logger := logr.FromContextOrDiscard(ctx).WithValues("checkForMissingInputs: synthesizer access", comp.Generation)
```

Comment:
This log column would be pretty noisy - remove it?

Reply:
So how would we indicate that the thing returned true only because the synthesizer wasn't found?